src/binary/ibc-setup/commands/init.spec.ts

```diff
@@ -0,0 +1,89 @@
+import fs from 'fs';
+
+import test from 'ava';
+import axios from 'axios';
+import sinon from 'sinon';
+
+import { Options, run } from './init';
+
+const consoleLog = sinon.stub(console, 'log');
+const fsExistSync = sinon.stub(fs, 'existsSync');
+const fsMkdirSync = sinon.stub(fs, 'mkdirSync');
+const axiosGet = sinon.stub(axios, 'get');
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const fsWriteFileSync = sinon.stub(fs, 'writeFileSync');
+const fsLstatSync = sinon.stub(fs, 'lstatSync');
+
+test.beforeEach(() => {
+  sinon.reset();
+
+  fsLstatSync.returns({
+    ...new fs.Stats(),
+    isDirectory: () => true,
+    isFile: () => true,
+  });
+});
+
+test('read existing registry.yaml', async (t) => {
+  const options: Options = {
+    home: '/home/user',
+    src: 'AAA',
+    dest: 'BBB',
+  };
+
+  fsExistSync.onCall(0).returns(true).onCall(1).returns(true);
+
+  const registryPath = `${options.home}/registry.yaml`;
+  const registryYaml = `
+  version: 1
+  `;
+
+  axiosGet.resolves({
+    data: registryYaml,
+  });
+  fsReadFileSync.returns(registryYaml);
+  fsWriteFileSync.returns(undefined);
+
+  await run(options);
+
+  t.assert(fsExistSync.calledWithExactly(options.home));
+  t.assert(fsExistSync.calledWithExactly(registryPath));
```

Comment:
I'm not sure it's necessary to check these calls. We want to make sure that eg network requests aren't being made or directories aren't being created, but is it important whether or not `fs.existsSync` is called?

Reply:
I agree it's not important. I'm going to remove that so the whole test is more readable.

5198dee